content/pt/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,258 @@
+
+1. Usando o comando `kubectl create` para criar um Deployment que gerencia um Pod. O Pod executa um contêiner baseado na imagem docker disponibilizada.
+
+    ```shell
+    kubectl create deployment hello-node --image=k8s.gcr.io/echoserver:1.4
+    ```
+
+2. Visualizando o Deployment:
+
+    ```shell
+    kubectl get deployments
+    ```
+
+    A saída será semelhante a:
+
+    ```
+    NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+    hello-node   1/1     1            1           1m
+    ```
+
+3. Visualizando o Pod:
+
+    ```shell
+    kubectl get pods
+    ```
+
+    A saída será semelhante a:
+
+    ```
+    NAME                          READY     STATUS    RESTARTS   AGE
+    hello-node-5f76cf6ccf-br9b5   1/1       Running   0          1m
+    ```
+
+4. Visualizando os eventos do cluster:
+
+    ```shell
+    kubectl get events
+    ```
+
+5. Visualizando a configuração do `kubectl`:
+
+    ```shell
+    kubectl config view
+    ```
+
+{{< note >}}
+Para mais informações sobre o comando `kubectl`, veja o [kubectl overview](/docs/reference/kubectl/overview/).
+{{< /note >}}
+
+## Criando um serviço
+
+Por padrão, um Pod só é acessível utilizando o seu endereço IP interno no cluster Kubernetes. Para dispobiblilizar o contêiner `hello-node` fora da rede virtual do Kubernetes, você deve expor o Pod como um [*serviço*](/docs/concepts/services-networking/service/) Kubernetes.
+
+1. Expondo o Pod usando o comando `kubectl expose`:
+
+    ```shell
+    kubectl expose deployment hello-node --type=LoadBalancer --port=8080
+    ```
+
+    O parâmetro `--type=LoadBalancer` indica que você deseja expor o seu serviço fora do cluster Kubernetes.
+    
+    A aplicação dentro da imagem `k8s.gcr.io/echoserver` "escuta" apenas na porta TCP 8080. Se você usou
+     `kubectl expose` para expor uma porta diferente, os clientes não conseguirão se conectar a essa outra porta.
+
+2. Visualizando o serviço que você acabou de criar:
+
+    ```shell
+    kubectl get services
+    ```
+
+    A saída será semelhante a:
+
+    ```
+    NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    hello-node   LoadBalancer   10.108.144.78   <pending>     8080:30369/TCP   21s
+    kubernetes   ClusterIP      10.96.0.1       <none>        443/TCP          23m
+    ```
+
+    Em provedores de Cloud que fornecem serviços de balaceamento de carga para o Kubernetes, um IP externo seria provisionado para acessar o serviço. No Minikube, o tipo `LoadBalancer` torna o serviço acessível por meio do comando `minikube service`.
```

Comment:
Please, can you fix the word "balaceamento" to "balanceamento"?
Thanks Jailton!